csharp/IceStorm/Weather/Station/Program.cs

```diff
@@ -11,9 +11,11 @@
 // listens on an OS-assigned TCP port, on all interfaces.
 Ice.ObjectAdapter adapter = communicator.createObjectAdapterWithEndpoints("StationAdapter", "tcp");
 
-// Register the ConsolePrinter servant with the adapter, and get a proxy to the new object. We use a UUID for the
-// identity because these subscribers (weather stations) are transient: when a subscriber shuts down, we want IceStorm
```

Comment:
This was confusing. IceStorm doesn't auto cleanup when a subscriber shuts down. It removes subscribers when it fails to deliver an event (based on the Retry QoS value), which is definitely not clear when you read the sentence above.